<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer</class>
 <widget class="QMainWindow" name="ImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Heightmap</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>514</width>
         <height>514</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_as"/>
   <addaction name="actionProperties"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="actionFit_to_window"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPan"/>
   <addaction name="actionExtrude"/>
   <addaction name="actionIntrude"/>
  </widget>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/media/24x24/document-save-as.png</normaloff>:/media/24x24/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/media/24x24/view-zoom-in.png</normaloff>:/media/24x24/view-zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/media/24x24/view-zoom-out.png</normaloff>:/media/24x24/view-zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionFit_to_window">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/media/24x24/view-zoom-reset.png</normaloff>:/media/24x24/view-zoom-reset.png</iconset>
   </property>
   <property name="text">
    <string>Fit to window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/media/24x24/document-properties.png</normaloff>:/media/24x24/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/media/24x24/cursor-select.png</normaloff>:/media/24x24/cursor-select.png</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
  <action name="actionExtrude">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/media/24x24/go-up.png</normaloff>:/media/24x24/go-up.png</iconset>
   </property>
   <property name="text">
    <string>Extrude</string>
   </property>
  </action>
  <action name="actionIntrude">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/media/24x24/go-down.png</normaloff>:/media/24x24/go-down.png</iconset>
   </property>
   <property name="text">
    <string>Intrude</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSave_as</sender>
   <signal>triggered()</signal>
   <receiver>ImageViewer</receiver>
   <slot>saveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionProperties</sender>
   <signal>triggered()</signal>
   <receiver>ImageViewer</receiver>
   <slot>showProperties()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_in</sender>
   <signal>triggered()</signal>
   <receiver>ImageViewer</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_out</sender>
   <signal>triggered()</signal>
   <receiver>ImageViewer</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFit_to_window</sender>
   <signal>triggered()</signal>
   <receiver>ImageViewer</receiver>
   <slot>resetZoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>zoomIn()</slot>
  <slot>zoomOut()</slot>
  <slot>resetZoom()</slot>
  <slot>saveAs()</slot>
  <slot>showProperties()</slot>
 </slots>
</ui>
